extensions/ql-vscode/.storybook/preview.js

```diff
@@ -0,0 +1,38 @@
+import { themes } from '@storybook/theming';
+import { action } from '@storybook/addon-actions';
+
+// Allow all stories/components to use Codicons
+import '@vscode/codicons/dist/codicon.css';
+
+import '../src/stories/vscode-theme.css';
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+  docs: {
+    theme: themes.dark,
+  },
+  backgrounds: {
+    default: 'dark',
+    values: [
+      {
+        name: 'dark',
+        value: '#1e1e1e',
+      },
+    ],
+  },
+  options: {
+    storySort: {
+      order: ['WebView UI Toolkit', 'MRVA'],
```

Comment:
Can we just have one set of stories (storybook?) and call it something generic? The component library we've started building with remote queries/variant analysis is meant to be used for local queries too eventually so it would be cool to avoid calling anything MRVA etc. Appreciate the current repo structure has nothing to imply that though.

Reply:
👍, I'll remove the MRVA prefix from the existing stories such that all components are just listed as top-level stories.